tests/dummydata.py

```diff
@@ -0,0 +1,316 @@
+#code implementing the tests variables of each class
+
+from wifi import subprocess_compat as subprocess
+from wifi.scheme import extract_schemes, Scheme
+from wifi.scan import Cell
+import string
+
+class ScanTesting(object):
+    args_ws = ['sudo', '/sbin/iwlist', 'interface', 'scan']
```

Comment:
the tests don't use this very often, I don't think that putting this data in here is really worth it.